content/en/about/feature-stages/index.md

```diff
@@ -69,18 +69,18 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Stable
-| [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/)        | Stable
-| [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/citadel-config/plugin-ca-cert/)        | Stable
 | [Service-to-service mutual TLS](/docs/concepts/security/#mutual-tls-authentication)         | Stable
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
+| [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/citadel-config/plugin-ca-cert/)        | Stable
+| [Authorization](/docs/concepts/security/#authorization)   | Beta
+| [End User (JWT) Authentication](/docs/concepts/security/#authentication)  | Beta
+| [Automatic mutual TLS](/docs/tasks/security/authentication/auto-mtls/) | Beta
 | [VM: Service Credential Distribution](/docs/concepts/security/#pki)         | Beta
-| [Mutual TLS Migration](/docs/tasks/security/authentication/mtls-migration)    | Beta
 | [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Beta
-| [Authorization](/docs/concepts/security/#authorization)   | Beta
-| [End User (JWT) Authentication](/docs/concepts/security/#authentication)  | Alpha
-| [OPA Checker](/docs/reference/config/policy-and-telemetry/adapters/opa/)    | Alpha
-| [SDS Integration](/docs/tasks/security/citadel-config/auth-sds/) | Alpha
```

Comment:
I don't see an item for `SDS Integration` any longer. Should it be removed, or was that just missed?

Reply:
Yes, can we add:
[SDS based key and certificate provisioning](/docs/concepts/security/#node-agent-in-kubernetes)  | Stable

Reply:
Updated.